### ISSUE TITLE
刚刚更新打最新版依旧黑屏

### ISSUE THREAD

**Issue by RealSlakey** [Cateners/tiny_computer#159]
黑屏但是可显示鼠标指针![Screenshot_2024-11-11-01-27-13-264_com.fct.tiny.jpg](https://github.com/user-attachments/assets/f369d605-ae9f-4029-8ba2-8874f8af4dce)

**Comment by jiushier** [comment]
同黑屏但可显示鼠标指针

**Comment by jiushier** [comment]
而且x11默认开启  无法关闭

**Comment by Cateners** [comment]
既然这样那只能把lxqt再加回去了。。。
可以说下机型吗？

**Comment by Cateners** [comment]
> 而且x11默认开启  无法关闭

你应该是下载到gxde的版本了

**Comment by RealSlakey** [comment]
> 既然这样那只能把lxqt再加回去了。。。
> 可以说下机型吗？

机型是小米MIX FOLD1

**Comment by jiushier** [comment]
> > 而且x11默认开启  无法关闭
> 
> 你应该是下载到gxde的版本了

是的 装了gxde版本后来换了win主题版本  上一版本是x11连接断开  这个版本可以连接但是如楼主说的黑屏只有鼠标  lxqt版本一直没用过

**Comment by hwlazy** [comment]
天玑处理器gxde版开启图形加速会黑屏，
